eowyn:
  pid_controller_node:
    ros__parameters:
      p_linear_gain   : 0.75
      p_angular_gain  : 0.50
